src/rtdb.ts

```diff
@@ -0,0 +1,47 @@
+import { Client } from "./apiv2";
+import { DatabaseInstance, populateInstanceDetails } from "./management/database";
+import { FirebaseError } from "./error";
+import { Options } from "./options";
+import { realtimeOriginOrCustomUrl } from "./database/api";
+import * as utils from "./utils";
+
+/**
+ * Updates rules, optionally specifying a dry run flag for validation purposes.
+ */
+export async function updateRules(
+  projectId: string,
+  instance: string,
+  src: any,
+  options: { dryRun?: boolean } = {}
+): Promise<void> {
+  let path = ".settings/rules.json";
+  if (options.dryRun) {
+    path += "?dryRun=true";
```

Comment:
Since we're moving to apiv2, consider switching this to use ClientRequestOptions.queryParams instead of just appending them to path

Reply:
Easy enough. Done.